pkg/controller/provider/validation.go

```diff
@@ -42,14 +44,15 @@ const (
 //
 // Reasons
 const (
-	NotSet       = "NotSet"
-	NotFound     = "NotFound"
-	NotSupported = "NotSupported"
-	DataErr      = "DataErr"
-	Malformed    = "Malformed"
-	Completed    = "Completed"
-	Tested       = "Tested"
-	Started      = "Started"
+	NotSet             = "NotSet"
+	NotFound           = "NotFound"
+	NotSupported       = "NotSupported"
+	DataErr            = "DataErr"
+	Malformed          = "Malformed"
+	Completed          = "Completed"
+	Tested             = "Tested"
+	Started            = "Started"
+	InsecureSkipVerify = "insecureSkipVerify"
```

Comment:
I think "SkipTLSVerification" for the condition reason instead of "InsecureSkipVerify" would be clearer.

Reply:
The reason for choosing `InsecureSkipVerify` was to keep the name used by the client, but I see the point here as it is a user facing string, makes sense to make it clearer to a user reading the condition, changing as suggested